### PR TITLE
add collector benchmark page

### PR DESCRIPTION
**Pull request by codeboten**
This is based on the work that @mwear started in the collector repository. Rather than publishing a github page from the collector repo, I suggest this should live in the opentelemetry.io repository and be displayed on the website as well.

For reviewers, is including style/html in the markdown ok? Or should this be moved somewhere else?

---

**Preview**: https://deploy-preview-2975--opentelemetry.netlify.app/docs/collector/benchmarks/

### CODE REVIEW

File: content/en/docs/collector/benchmarks.md [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+---
+  <style>
+
+    header {
+      width: 100%;
+      display: flex;
+      flex-direction: row;
+      background-color: rgb(79, 98, 173);
+    }
+
```
Comment (mwear) [inline content/en/docs/collector/benchmarks.md:13]: We removed the header from the html, we can probably delete this and rely on the header css for opentelemetry.io

```suggestion
```

File: content/en/docs/collector/benchmarks.md [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+    .logo svg {
+      height: 48px;
+      margin-left: 30px;
+    }
+
+    .header-item {
+      flex: 1;
+    }
+
```
Comment (mwear) [inline content/en/docs/collector/benchmarks.md:87]: These too

```suggestion
```

**Review by mwear — COMMENTED** [review]
I think we can delete some CSS, but generally this looks good to me.

**Comment by austinlparker** [comment]
cc @chalin -- thoughts on this? some wonkiness with breakpoints due to max-width looks like.

**Comment by austinlparker** [comment]
Couple of other notes -

- Would be great to have an index/TOC here, it's a _lot_ of charts.
- Would be great to have explanations/descriptions of the benchmarks
- Is there a way we can consolidate this to top-k (10?) critical benchmarks for web and put the rest in the repo?

**Comment by chalin** [comment]
@austinlparker - once your concerns are addressed (which seem reasonable to me -- e.g. top-x), I can certainly look at integrating the page. Actually, it could always land with embedded CSS etc. and be refactored after.

**Comment by jpkrohling** [comment]
I came here for the same comments @austinlparker has already made :-) An intro section setting the context (is it about the contrib image? Core? both?) and a short description of the scenarios would be required.

**Comment by codeboten** [comment]
I've added a paragraph at the top, PTAL

**Review by mwear — APPROVED** [review]

**Review by cartermp — APPROVED** [review]

**Pull request merged**